zerver/webhooks/yo/doc.md

```diff
@@ -0,0 +1,28 @@
+See your Yo App notifications in Zulip!
+
+First, on your {{ settings_html|safe }}, create a bot for
```

Comment:
We should probably use `templates/zerver/help/include/create-bot-construct-url.md` in here.

Maybe with some rephrasing/structural changes we can make the "Next" at the beginning of the template fit? Another fix could be removing the nexus (i.e. next) from the template.